models/players.js

```diff
@@ -223,7 +243,7 @@ function mongoInsertPlayers(rb_team_id, league, team_document){
   console.log("inserting into the DB");
   db.open(function(err, db){
     db.collection("players").update({team_id: rb_team_id},
-    {$set: {team_id: team_document["team_id"], team_name: team_document["team_name"], league: league, last_updated: new Date().toISOString().slice(0, 19).replace('T', ' '), players: team_document["players"]}},
+    {$set: {team_id: team_document["team_id"], team_name: team_document["team_name"], league: league, last_updated: new Date().toISOString().slice(0, 19).replace('T', ' '), starters: team_document.starters, bench: team_document.bench}},
```

Comment:
Can we be consistent with either team_document["starters"] or team_document.starters with this list?